android/src/main/java/io/sentry/RNSentryModule.java

```diff
@@ -107,13 +111,11 @@ public void startWithOptions(final ReadableMap rnOptions, Promise promise) {
                 return event;
             });
 
-            if (rnOptions.hasKey("enableNativeCrashHandling")
-                    && !rnOptions.getBoolean("enableNativeCrashHandling")) {
+            if (rnOptions.hasKey("enableNativeCrashHandling") && !rnOptions.getBoolean("enableNativeCrashHandling")) {
                 final List<Integration> integrations = options.getIntegrations();
                 for (final Integration integration : integrations) {
-                    if (integration instanceof UncaughtExceptionHandlerIntegration ||
-                            integration instanceof AnrIntegration ||
-                            integration instanceof NdkIntegration) {
+                    if (integration instanceof UncaughtExceptionHandlerIntegration
```

Comment:
Linter kicked in here when I saved, nothing changed here except styling.
I can revert if it's making reviewing harder